epochtalk/epochtalk
# Deletion

User Deletion

Testing Procedures:
* Create a new account 
* Create a thread and add a few posts
* Delete a few posts and ensure they are not publicly visible
* Go to the profile page and deactivate the user
* Ensure that all posts by the user is not publicly visible
* Reactivate the user
* Deactivate the user using an admin account
* Ensure user is deactivated and posts are not visible
* Delete the user using an admin account
* Ensure all the posts and thread by the user are no longer available

To be tested with the deletion branch in epochtalk/core-pg
